ci_framework/roles/edpm_build_images/defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_edpm_build_images"
+
+cifmw_edpm_build_images_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_edpm_build_images_artifacts: "{{ cifmw_edpm_build_images_basedir }}/artifacts"
+cifmw_edpm_build_images_via_rpm: true
+cifmw_build_host_packages:
+  - diskimage-builder
+  - openstack-ironic-python-agent-builder
+  - buildah
+cifmw_edpm_image_builder_repo_path: >-
+  {%- if cifmw_edpm_build_images_via_rpm -%}
+    /usr/share/edpm-image-builder/
+  {%- else -%}
+    {{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/edpm-image-builder
+  {%- endif -%}
+cifmw_edpm_build_images_elements:
+  - "/usr/share/ironic-python-agent-builder/dib/"
+  - "{{ cifmw_edpm_image_builder_repo_path }}/dib/"
+cifmw_edpm_build_images_dib_local_image_path: "{{ cifmw_edpm_build_images_basedir }}/artifacts/edpm_images/"
```

Comment:
Downloading image in artifcats is bad idea, because we capture artifacts that means we capture 1 gb of image. lets download in home directory

Reply:
removed, we now download in home directory